guides/source/routing.md

```diff
@@ -1115,6 +1115,11 @@ You may restrict the listing to the routes that map to a particular controller s
 $ CONTROLLER=users bin/rake routes
 ```
 
+Also you may grep your routers with `-g` options. This option returns the url `name`, `verb`, `path` and `reqs` field that match the pattern:
```

Comment:
routers should be routes. I'd remove options after `-g`.

You make it sound like we return only the part of the route that matches - the name or verb etc. - instead of the whole route. I'm also unsure if we need to explain `grep`-ing. I think it makes the documentation clearer if we assume the reader knows that.
So how about: "You can grep your routes with `-g`, which filters routes by `name`, `verb`, `path`, or `reqs`."?